rpc-interface/src/consensus.rs

```diff
@@ -138,7 +138,7 @@ pub trait ConsensusInterface {
         wallet: Address,
         contract_address: Address,
         recipient: Address,
-        pre_image: AnyHash,
+        pre_image: PreImage,
```

Comment:
So do I understand the deserializer for `PreImage` correctly, that in the JSON-RPC call just a regular HEX string is sent for the `pre_image` field?

Reply:
Exactly!